### DISCUSSION TITLE
Some suggestions on the build

### DISCUSSION THREAD

**Discussion by sunnychanwork** [microsoft/openjdk#12] [Q&A] [answered]
In the Release notes section it would be useful to state the release version number with the "microsoft specific enhancement". For example, if I run java -version and see "Microsoft-18724" build number I should be easily find what has changed in that build.

Also would it be possible to expose the repo in github so I can see for myself the specific changes in the build, instead of looking through the source tarball?

Also I noticed that Shenandoah is enabled which is optional in Java 11 - it would be useful to document any changes deviated from the standard build.

**Comment by karianna** [comment]
@sunnychanwork - Nice to hear from you!

We can and will work on all three of those - both the GitHub repo and the note on Shenandoah are already planned for and I like your suggestion on the revision number indicating the patch (we may go with a commit hash as well).

**Comment by karianna** [comment]
FYI - Shenandoah is a default supported GC in OpenJDK 11.  I believe that only Oracle chooses to omit it, but we'll add something to the docs for clarity.

**Comment by karianna** [comment, marked as answer]
The GitHub repos are done!  see [microsoft/openjdk-jdk11u](https://github.com/microsoft/openjdk-jdk11u) as an example
